code/modules/power/cable.dm

```diff
@@ -556,14 +556,14 @@ By design, d1 is the smallest direction and d2 is the highest
 		new_cable.update_icon()
 
 /obj/item/stack/cable_coil/attack_self(mob/user)
-	if(!use(15))
-		to_chat(user, "<span class='notice'>You dont have enough cable coil to make restraints out of them</span>")
+	if(amount < 15)
+		to_chat(user, "<span class='notice'>You don't have enough cable coil to make restraints out of them</span>")
 		return
 	to_chat(user, "<span class='notice'>You start making some cable restraints.</span>")
 	if(!do_after(user, 30, TRUE, user, TRUE))
```

Comment:
```suggestion
	if(!do_after(user, 30, TRUE, user, TRUE) || !use(15))
```

Reply:
Wont that make the cables get used during the do_after?

Reply:
Nevermind, tested it and it didnt